src/components/ListingItem.tsx

```diff
@@ -8,13 +8,11 @@ type ListingItemProps = {
 export class ListingItem extends React.Component<ListingItemProps> {
   render() {
     return (
-      <li>
-        <label>
-          {this.props.item.id}
-          {this.props.item.url}
-          {this.props.item.date}
-        </label>
-      </li>
+      <tr>
```

Comment:
How about also adding a row of <th/> tags if switching to <table/> approach?

Reply:
Cool, I will add <tbody />.